HPXMLtoOpenStudio/measure.rb

```diff
@@ -2807,6 +2808,25 @@ def self.add_ceiling_fans(runner, model, weather)
                             @cfa, @living_space)
   end
 
+  def self.add_dehumidifier(runner, model)
+    return if @hpxml.dehumidifiers.size == 0
+
+    dehumidifier = @hpxml.dehumidifiers[0]
+    sys_id = dehumidifier.id
+    @hvac_map[sys_id] = []
+    water_removal_rate = dehumidifier.capacity
+    energy_factor = dehumidifier.energy_factor
+    integrated_energy_factor = dehumidifier.integrated_energy_factor
+    humidity_setpoint = dehumidifier.rh_setpoint
+    fraction_served = dehumidifier.fraction_served
+
+    # Calculate air flow rate by assuming 2.75 cfm/pint/day (based on experimental test data)
+    air_flow_rate = 2.75 * water_removal_rate
+
+    HVAC.apply_dehumidifier(model, runner, energy_factor, integrated_energy_factor, water_removal_rate,
```

Comment:
Now that we have HPXML objects, how about just passing the `dehumidifier` object as the argument to `apply_dehumidifier`? Then we don't have all of these local variables doing hardly anything. The air_flow_rate calc can also go inside the `apply_dehumidifier` method. I('m going to start moving more of our code to this approach of passing objects when it makes sense.)